README.md

```diff
@@ -61,6 +61,11 @@ On OpenSUSE, it can be installed with:
 sudo zypper install readline-devel
 ```
 
+On ArchLinux, it can be installed with:
```

Comment:
> the commit message type (in this case `feat`) should be `docs` instead, since its just adding documentation

oh that i hesitated, i had read it but the feat was automatic forgiveness